server/microservices/auth/index.ts

```diff
@@ -1,11 +1,11 @@
 import express, { Express, Request, Response } from 'express';
-import dotenv from 'dotenv';
-
-dotenv.config();
+import findConfig from 'find-config';
+import * as dotenv from 'dotenv';
+dotenv.config({ path: findConfig('.env') ?? undefined });
 
 const app: Express = express();
 app.disable('x-powered-by');
-const port = process.env.PORT || 3001;
+const port = process.env.AUTH_PORT || 3001;
```

Comment:
```suggestion
const port = process.env.PORT || 3001;
```
We will deploy all of the microservices in the separate docker containers. That's why we don't need to clarify this env variable

Reply:
Got it